demisto_sdk/commands/content_graph/objects/base_content.py

```diff
@@ -21,3 +51,7 @@ class Config:
 
     def to_dict(self) -> Dict[str, Any]:
         return json.loads(self.json())
+
+    @abstractmethod
+    def dump(self, path: DirectoryPath, marketplace: MarketplaceVersions) -> None:
```

Comment:
Is `marketplace` needed? It should be set in the os.env.

Reply:
let's keep it we would like in the future to run one script for both marketplaces